x-pack/plugins/ml/public/application/components/data_grid/data_grid.tsx

```diff
@@ -90,6 +95,43 @@ export const DataGrid: FC<Props> = memo(
     //   };
     // };
 
+    const popOverContent = useMemo(() => {
+      return analysisType === ANALYSIS_CONFIG_TYPE.REGRESSION ||
+        analysisType === ANALYSIS_CONFIG_TYPE.CLASSIFICATION
+        ? {
+            featureImportance: ({ children }: { cellContentsElement: any; children: any }) => {
+              const rowIndex = children?.props?.visibleRowIndex;
+              const row = data[rowIndex];
+              if (!row) return <div />;
+              const parsedFIArray = row.ml.feature_importance;
+              let predictedValue: string | number | undefined;
+              let topClasses: TopClasses = [];
+              if (
+                predictionFieldName !== undefined &&
+                row &&
+                row.ml[predictionFieldName] !== undefined
+              ) {
+                predictedValue = row.ml[predictionFieldName];
+                topClasses = row.ml.top_classes;
+              }
```

Comment:
`ml` is hard-coded here but it could be configured to be any other value using a custom value defined in the config under `dest.results_field` https://www.elastic.co/guide/en/elasticsearch/reference/7.9/put-dfanalytics.html

Reply:
Good catch! Thanks so much. Will update.

Reply:
Updated here https://github.com/elastic/kibana/pull/73561/commits/ef815d11a2bf334d703366daff3519b38b848e11